apps/ocean-api/__tests__/controllers/RawTxController.test.ts

```diff
@@ -1,40 +1,118 @@
 import { OceanApiTesting } from '../../testing/OceanApiTesting'
+import { Elliptic } from '@defichain/jellyfish-crypto'
+import { ApiException } from '@defichain/ocean-api-client'
 
 const apiTesting = OceanApiTesting.create()
+const privA = Elliptic.fromPrivKey(Buffer.from('619c335025c7f4012e556c2a58b2506e30b8511b53ade95ea316fd8c3286feb9', 'hex'))
+const privB = Elliptic.fromPrivKey(Buffer.from('557c4bdff86e59015987c1c7f3328a1fb4c2177b5e834f09c8cd10fae51af93b', 'hex'))
 
-beforeEach(async () => {
-  await apiTesting.start()
-})
+describe('feeEstimate', () => {
+  beforeEach(async () => {
+    await apiTesting.start()
+  })
 
-afterEach(async () => {
-  await apiTesting.stop()
-})
+  afterEach(async () => {
+    await apiTesting.stop()
+  })
 
-it('should be fixed fee of 0.00005000 when there are no transactions', async () => {
-  const feeRate = await apiTesting.client.rawtx.feeEstimate()
+  it('should be fixed fee of 0.00005000 when there are no transactions', async () => {
+    const feeRate = await apiTesting.client.rawtx.feeEstimate()
 
-  expect(feeRate).toStrictEqual(0.00005000)
-})
+    expect(feeRate).toStrictEqual(0.00005000)
+  })
 
-it('should have fee of not 0.00005 with transactions activity', async () => {
-  await apiTesting.container.waitForWalletBalanceGTE(100)
+  it('should have fee of not 0.00005 with transactions activity', async () => {
+    await apiTesting.container.waitForWalletBalanceGTE(100)
 
-  for (let i = 0; i < 10; i++) {
-    for (let x = 0; x < 20; x++) {
-      await apiTesting.rpc.wallet.sendToAddress('bcrt1qf5v8n3kfe6v5mharuvj0qnr7g74xnu9leut39r', 0.1, {
-        subtractFeeFromAmount: true,
-        avoidReuse: false
-      })
+    for (let i = 0; i < 10; i++) {
+      for (let x = 0; x < 20; x++) {
+        await apiTesting.rpc.wallet.sendToAddress('bcrt1qf5v8n3kfe6v5mharuvj0qnr7g74xnu9leut39r', 0.1, {
+          subtractFeeFromAmount: true,
+          avoidReuse: false
+        })
+      }
+      await apiTesting.container.generate(1)
     }
-    await apiTesting.container.generate(1)
-  }
 
-  const feeRate = await apiTesting.client.rawtx.feeEstimate()
-  expect(feeRate).not.toStrictEqual(0.00005000)
+    const feeRate = await apiTesting.client.rawtx.feeEstimate()
+    expect(feeRate).not.toStrictEqual(0.00005000)
```

Comment:
newbie here. im wondering if it would be better to expect the fee to be greater than 0.00005000 when there are transactions?

Reply:
min fee 0.00005 is a fixed rate
need to tight strictly using == instead of >, >=, <=, <
if ain updates/or mistakenly update the min fee rate.. then we know

Reply:
ar in this case cos adding a tx will increase the fee from the base tx. so im wondering if > would be better?

Reply:
the test is testing the min fee should be 0.00005 whilst no txns

Reply:
gosh.. i was referring wrong statement..  ignore my comment above :sweat_smile: 
in this case.. using `>=` is more accurate

Reply:
the min fee rate is `0.00001` set on ain
BTW, the `0.00005` is hardcoded when no txs, its originally `{ errors: [ 'Insufficient data or no feerate found' ], blocks: 0 }`
the fee could be lesser than `0.00005` but greater than `0.00001`
the test applied `!= 0.00005` as its not estimated

@fuxingloh kindly correct me if that's something not right on my checking

Reply:
thanks for clearing the air